x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/setup_page/index.tsx

```diff
@@ -38,84 +67,154 @@ export const SetupPage: React.FunctionComponent<{
-          {space}- Enable Elasticsearch API keys.
-          {space}- Enable TLS to secure the communication between Agents and Kibana.
-          {space}- Set the encryption key for encrypted saved objects.
-          "
-            values={{
-              space: <EuiSpacer size="m" />,
-            }}
+            id="xpack.ingestManager.setupPage.missingRequirementsElasticsearchTitle"
+            defaultMessage="In your Elasticsearch configuration, enable:"
           />
-        </EuiText>
-        <EuiSpacer size="l" />
-      </>
-    ) : (
-      <>
-        <EuiSpacer size="m" />
-        <EuiIcon type="lock" color="subdued" size="xl" />
-        <EuiSpacer size="m" />
-        <EuiTitle size="l">
-          <h2>
+          <EuiSpacer size="l" />
+          <RequirementItem isMissing={false}>
+            <FormattedMessage
+              id="xpack.ingestManager.setupPage.elasticsearchSecurityFlagText"
+              defaultMessage="Elasticsearch security. Set {securityFlag} to {true} ."
+              values={{
+                securityFlag: <EuiCode>xpack.security.enabled</EuiCode>,
+                true: <EuiCode>true</EuiCode>,
+              }}
+            />
+          </RequirementItem>
+          <EuiSpacer size="s" />
+          <RequirementItem isMissing={missingRequirements.includes('api_keys')}>
             <FormattedMessage
-              id="xpack.ingestManager.setupPage.enableTitle"
-              defaultMessage="Enable Fleet"
+              id="xpack.ingestManager.setupPage.elasticsearchApiKeyFlagText"
+              defaultMessage="API key service. Set {apiKeyFlag} to {true} ."
+              values={{
+                apiKeyFlag: <EuiCode>xpack.security.authc.api_key.enabled</EuiCode>,
+                true: <EuiCode>true</EuiCode>,
+              }}
             />
-          </h2>
-        </EuiTitle>
-        <EuiSpacer size="xl" />
-        <EuiText color="subdued">
+          </RequirementItem>
+          <EuiSpacer size="m" />
+          <EuiCodeBlock isCopyable={true}>
+            {`xpack.security.enabled: true
+xpack.security.authc.api_key.enabled: true`}
+          </EuiCodeBlock>
+          <EuiSpacer size="l" />
           <FormattedMessage
-            id="xpack.ingestManager.setupPage.enableText"
-            defaultMessage="In order to use Fleet, you must create an Elastic user. This user can create API keys
-        and write to logs-* and metrics-*."
+            id="xpack.ingestManager.setupPage.missingRequirementsKibanaTitle"
+            defaultMessage="In your Kibana configuration, enable:"
+          />
+          <EuiSpacer size="l" />
+          <RequirementItem isMissing={missingRequirements.includes('tls_required')}>
+            <FormattedMessage
+              id="xpack.ingestManager.setupPage.tlsFlagText"
+              defaultMessage="Kibana security. Set. Set {securityFlag} to {true}. For development purposes, you can disable TLS by setting {tlsFlag} to {true} as an unsafe alternative."
```

Comment:
in designs, TLS is linked.maybe it's meant to link to https://www.elastic.co/guide/en/kibana/current/configuring-tls.html?